src/utils/text.test.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  capitalizeFirst,
+  fitText,
+  formatListToHuman,
+  handlePlural,
+} from "./text";
+
+describe("text", () => {
+  describe("fitText", () => {
+    it("should return the string if it is shorter than the length", () => {
+      expect(fitText("hello", 10)).toBe("hello");
+    });
+
+    it("should return the string if it is equal to the length", () => {
+      expect(fitText("hello", 5)).toBe("hello");
+    });
+
+    it("should return the string if it is shorter than the length and includeTrail is false", () => {
+      expect(fitText("hello", 10, false)).toBe("hello");
+    });
```

Comment:
This should rather test with 4 as length, aka. `fitText("hello", 4, false)` and should be equal to `hell`. This toggle doesn't really do anything unless it overflows.

Reply:
Thanks for the review :)

The test was exactly to make sure that the string was not modified if shorter than the max.
However, a test for the use case you just mentioned is indeed missing, I'll add it now.